agent/agent.go

```diff
@@ -395,6 +401,21 @@ func (a *agent) initSocketServer() {
 	a.logger.Debug(a.hardCtx, "socket server started", slog.F("path", a.socketPath))
 }
 
+// startBoundaryLogProxyServer starts the boundary log proxy socket server.
+func (a *agent) startBoundaryLogProxyServer() {
+	const boundaryAuditSocketPath = "/tmp/boundary-audit.sock"
```

Comment:
This should use `os.TempDir`.

Reply:
I have parameterized this with an environment variable and a default path defined.